modules/installation-gcp-shared-vpc-permissions.adoc

```diff
@@ -0,0 +1,25 @@
+// This file is referenced in the following assembly:
+// installing/installing_gcp/installing-gcp-shared-vpc.adoc
+
+:_content-type: PROCEDURE
+[id="installation-gcp-shared-vpc-permissions_{context}"]
+= Required GCP permissions for shared VPC installation
+
+When you attach the `Owner` role to the service account that you create, you
+grant that service account all permissions, including those that are required to
+install {product-title}. To deploy an {product-title} cluster into a shared VPC, the
+service account requires the following permissions in the host project.
+
+.Required roles for the installation program
+* Compute Admin
+* Compute Load Balancer Services User
+
+.Required roles for creating network resources during installation
```

Comment:
1. Suggest to remove `DNS Administrator`, because in 4.13 private DNS zone & record sets and DNS record-sets in public DNS zone would be in **GCP service project**, and the only DNS related permission needed in **GCP host project** is `dns.networks.bindPrivateDNSZone`. 
2. It seems no pre-defined role of `Compute Firewall Create`. Instead, `Compute Security Admin (roles/compute.securityAdmin)` can be used if we want the installer creating firewall-rules in **GCP host project**. 

_FYI the binding roles in **GCP host project** of the service account I used for IPI XPN installation:_

**Scenario A:** the installer won't create firewall-rules in **GCP host project**
$ **gcloud projects get-iam-policy openshift-qe-shared-vpc --flatten='bindings[].members' --format='table(bindings.role)' --filter='bindings.members:ipi-xpn-minpt-permissions@openshift-qe.iam.gserviceaccount.com'**
ROLE
**projects/openshift-qe-shared-vpc/roles/dns.networks.bindPrivateDNSZone**
**roles/compute.networkUser**
$ **gcloud --project openshift-qe-shared-vpc iam roles describe dns.networks.bindPrivateDNSZone**
etag: BwXq5Yskvx4=
includedPermissions:
\- dns.networks.bindPrivateDNSZone
name: projects/openshift-qe-shared-vpc/roles/dns.networks.bindPrivateDNSZone
stage: ALPHA
title: dns.networks.bindPrivateDNSZone
$

**Scenario B:** the installer will create firewall-rules in **GCP host project**
$ **gcloud projects get-iam-policy openshift-qe-shared-vpc --flatten='bindings[].members' --format='table(bindings.role)' --filter='bindings.members:ipi-xpn-minpt-permissions@openshift-qe.iam.gserviceaccount.com'**
ROLE
**projects/openshift-qe-shared-vpc/roles/dns.networks.bindPrivateDNSZone**
**roles/compute.networkUser**
**roles/compute.securityAdmin**
$ **gcloud --project openshift-qe-shared-vpc iam roles describe dns.networks.bindPrivateDNSZone**
etag: BwXq5Yskvx4=
includedPermissions:
\- dns.networks.bindPrivateDNSZone
name: projects/openshift-qe-shared-vpc/roles/dns.networks.bindPrivateDNSZone
stage: ALPHA
title: dns.networks.bindPrivateDNSZone
$